database/migrations/20170531173912_organizations.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex =>
+  knex.schema.createTable( 'organizations', table => {
+    table.increments( 'id' ).primary()
+    table.integer( 'user_id' ).notNullable()
+    table.string( 'user_name' ).notNullable()
+    table.string( 'organization_name' ).notNullable()
```

Comment:
A little redundant to have a column called `organization_name` in the `organizations` table, since then you'll have to write SQL like this:

```sql
SELECT organization_name FROM organizations;
```

When I think it reads nicer to just have

```sql
SELECT name FROM organizations;
```

It's implied that the `name` is for the org, since it's within the table.